### PR TITLE
Code examples for episode 099 & 100

### PR DESCRIPTION
**Pull request by int3l**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -107,7 +107,7 @@ the full [playlist] is on youtube
 1. [typing: what is Type[X]? (intermediate)](https://youtu.be/_7EXU9cjBkg?list=PLWBKAf81pmOaP9naRiNAqug6EBnkPakvY)
 1. [python debugger crash course: pdb / breakpoint (beginner - intermediate)](https://youtu.be/0LPuG825eAk?list=PLWBKAf81pmOaP9naRiNAqug6EBnkPakvY)
 1. [why pytest.mark.usefixtures? (intermediate)](https://youtu.be/BE2v1VCmGwg?list=PLWBKAf81pmOaP9naRiNAqug6EBnkPakvY)
-1. [git apply and manual stashing (intermediate) -](https://youtu.be/4LXTaikZfco?list=PLWBKAf81pmOaP9naRiNAqug6EBnkPakvY)
```
Comment (asottile) [inline README.md:110]: oh, this file is auto generated -- let me just fix the actual video

**Review by asottile — APPROVED** [review]
can rebase out the extra commit -- the nightly job'll fix it automatically

**Comment by int3l** [comment]
Is it going to also fix the title in the specific sample README.md?

**Comment by asottile** [comment]
> 
> 
> Is it going to also fix the title in the specific sample README.md?

hmmm right -- you'll want to fix your README but not the main one

**Comment by int3l** [comment]
Done.

**Pull request merged**